src/parkapi_sources/converters/bahn_v2/mapper.py

```diff
@@ -44,3 +58,93 @@ def map_static_parking_site(bahn_input: BahnParkingSiteInput) -> StaticParkingSi
                 static_parking_site_input.capacity_disabled = int(round(capacity_data.total))
 
         return static_parking_site_input
+
+
+class BahnBikeParkingLockedMapper:
```

Comment:
That and the mapper below is almost a 100 % copy of the mapper above. In general, please try to avoid copy-pasting large parts of the code and do OOP. In this case, I wonder why you need different Mappers at all, as this could all be done in one source and one mapper.

Reply:
It's now updated.